raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -975,7 +975,7 @@ public void handleClaim(int epoch) {
         }
 
         @Override
-        public void handleResign() {
+        public void handleResign(int epoch) {
```

Comment:
as mentioned above, should this be a long?

Reply:
I actually think the Raft layer uses Int, so I'm thinking maybe this is correct and the use of Long elsewhere might be where a change could happen.

Reply:
this is in the raft layer so it's treated as 31 bits right now